pkgs/development/tools/stagit-gopher/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, libgit2, fetchgit }:
+
+stdenv.mkDerivation rec {
+  pname = "stagit-gopher";
+  version = "0.9.3";
+
+  src = fetchgit {
+    url = "git://git.codemadness.org/stagit-gopher";
+    rev = version;
+    sha256 = "0dcyi3k900jb8p5qhgfk91gvplblysgfz1vyn6pqrlk931hpac80";
+    };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  buildInputs = [ libgit2 ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://git.codemadness.org/stagit-gopher/file/README.html";
+    description = "Static git site generator for gopher";
+    license = licenses.mit;
+    platforms = platforms.all;
+    maintainers = with maintaininers; [ hcssmith ];
```

Typo here, causing ofborg to fail.
```suggestion
    maintainers = with maintainers; [ hcssmith ];
```

---

OK, I've corrected the typo, not sure how i missed that given that I wrote it two words before, and I have moved the } back two spaces which I think is what you meant.